bot/cogs/grades_data/grades_cog.py

```diff
@@ -189,6 +189,7 @@ def initialize(self, year):
             raise FileNotFoundError(not_found)
             
     def getFirstLast(self, Name):
+        # NEEDS IMPROVEMENTS (See 3x names (Yvon H Feaster) and/or 4x names (Rodger Larry Van Scoy))
```

Comment:
we should open an issue on this, can probably go hand in hand with the search command